docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
Sphinx?

Reply:
Did you mean to check this file in?

Reply:
http://www.sphinx-doc.org/en/master/

Reply:
Ha! I completely forgot about this project (which is sad, because I've used it in the past...)